submissions/expero1/html-css-popup/style.css

```diff
@@ -0,0 +1,397 @@
+  border: 1px solid grey;
+  border-radius: 5px;
+  box-shadow: 0 0 5px grey;
+  background-color: white;
+  z-index: 10;
+}
+
+.popup-checkbox:checked ~ .popup-menu {
+  display: block;
+}
+.main-menu a:checked ~ .popup-menu-wrapper .popup-menu,
+.main-menu a:hover ~ .popup-menu-wrapper .popup-menu,
+.main-menu a:focus ~ .popup-menu-wrapper .popup-menu,
+.user-menu a:checked ~ .popup-menu-wrapper > .popup-menu,
+.user-menu a:hover ~ .popup-menu-wrapper > .popup-menu,
+.user-menu a:focus ~ .popup-menu-wrapper > .popup-menu {
+  display: none;
+}
+.popup-menu-wrapper {
+  position: relative;
+  grid-area: popup-menu-wrapper;
+  z-index: 10;
+}
+
+.popup-menu-inner {
+  background-color: white;
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  justify-items: center;
+  padding: 5px;
+  gap: 20px 10px;
+}
+
+.popup-menu-inner.hidden {
+  display: none;
+}
+
+/* Product btn*/
+
+.product-btn {
+  display: inline-block;
+  list-style: none;
+  width: 6rem;
+  height: 6rem;
+  padding: 5px;
+}
+
+.product-btn:hover,
+.product-btn:focus {
+  outline: 2px solid grey;
+  box-shadow: 0 0 15px grey;
+}
+.popup-menu-inner li {
+  list-style: none;
+}
+.popup-menu-inner .product-btn {
+  display: flex;
+  height: 100%;
+  flex-direction: column;
+  justify-content: flex-end;
+  gap: 5px;
+  align-items: center;
+  background-color: white;
+  margin: 0 auto;
+  position: relative;
+}
+
+.product-btn img {
+  display: block;
+  object-fit: contain;
+  width: 3em;
+  height: 3em;
+  margin: 0 auto;
+}
+
+.product-btn span {
+  text-align: center;
+  display: block;
+  margin-top: 5px;
+}
+
+/* SHOW MORE button*/
+
+.show-more-checkbox {
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  appearance: none;
+  outline: none;
+  display: inline;
+}
+.btn {
+  width: 100%;
+  border: none;
+  height: 3rem;
+  padding: 1rem 0;
+  background-color: whitesmoke;
+  text-align: center;
+  text-transform: uppercase;
+  cursor: pointer;
+}
+.btn:hover,
+.btn:focus {
+  background-color: gray;
+}
+.btn:active {
+  background-color: dimgray;
+}
+.show-more-btn {
+  display: block;
+  text-transform: uppercase;
+}
+.show-more-checkbox:checked,
+.show-more-checkbox:checked ~ .show-more-btn {
+  display: none;
+}
+
+.show-more-checkbox:checked ~ .hor-line {
+  display: block;
+  position: relative;
+  margin: 10px 0 5px 0;
+  height: 1px;
+  border-bottom: 1px solid gray;
+  box-shadow: 0 0 10px gray;
+}
+.show-more-checkbox:checked ~ .popup-menu-inner.hidden {
+  display: grid;
+}
+.show-more-checkbox:focus ~ .show-more-btn {
+  background-color: gray;
+}
+
+.show-more-checkbox:checked ~ .even-more-btn {
+  display: block;
+}
+
+.even-more-btn {
+  width: 100%;
+  display: none;
+  margin-top: 10px;
+
+  border: none;
+  cursor: pointer;
+  height: 3em;
+}
+.even-more-btn:hover,
+.even-more-btn:focus {
+  background-color: gray;
+}
+
+.notification-menu {
+  grid-area: notification-menu;
+}
+.user-menu {
+  grid-area: user-menu;
+}
+.user-menu img,
```

Comment:
Avoid styling by HTML tags except if you add base styles. [Reason](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css)
Also, avoid using too specific CSS selectors. Read more about [CSS specificity](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity)

Reply:
Thank you. I've added classes for elements on the page and added class selectors to the CSS file for almost all selectors where I've used the HTML tag before.